CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 =======
+## [1.70.4] - 2023-08-31
+- logging: fix logged error in observability middleware when fields of transport.Request is in the tagsBlocklist
+- `go.mod`: update minimum requirements to go1.21 instead of go1.14 and update `golang.org/x/net v0.7.0` to v0.14.0
+- middleware stack usage: remove ~2KB of stack usage from the rpc handler function, 
```

Comment:
This part is under ##[Unreleased] before. I'm not sure if I should delete or keep it?
I see a [commit](https://github.com/yarpc/yarpc-go/commit/c14c5dc1cc53b6f083404508d4058eb371f2adfb) but it's not in 1.70.3 release note.